gammapy/makers/background/reflected.py

```diff
@@ -14,8 +15,33 @@
 
 log = logging.getLogger(__name__)
 
+FULL_CIRCLE = Angle(2 * np.pi, "rad")
+
+
+class RegionsFinder(metaclass=ABCMeta):
+    '''Baseclass for regions finders'''
+
+    @abstractmethod
+    def run(self, region, center):
+        """Find regions to calculate background counts.
+
+        Parameters
+        ----------
+        region : `~regions.SkyRegion`
+            Region to rotate
+        center : `~astropy.coordinates.SkyCoord`
+            Rotation point
+
+        Returns
```

Comment:
Maybe directly returning a list of `RegionGeom` objects would avoid returning `SkyRegion` and a `WCS`

Reply:
I discussed this with @adonath yesterday. Unfortunately, creating a `RegionGeom` created problems, as the number of found regions was not knowable by the maker then. 

The number of regions is however required for the alpha calculation.

Reply:
One could change the alpha calculation to the ratio of solid angles, but this can be easily done in a follow up PR.

Reply:
Is there an easy way to get this?

Reply:
```

In [4]: geom.solid_angle()
---------------------------------------------------------------------------
NotImplementedError                       Traceback (most recent call last)
~/Projects/gammapy/gammapy/makers/background/reflected.py in <module>
----> 1 geom.solid_angle()

~/Projects/gammapy/gammapy/maps/region/geom.py in solid_angle(self)
    302             raise ValueError("Region definition required.")
    303 
--> 304         area = self.region.to_pixel(self.wcs).area
    305         solid_angle = area * proj_plane_pixel_area(self.wcs) * u.deg ** 2
    306         return solid_angle.to("sr")

~/.local/anaconda/envs/gammapy-dev/lib/python3.7/site-packages/regions/core/compound.py in area(self)
    162     @property
    163     def area(self):
--> 164         raise NotImplementedError
    165 
    166     def rotate(self, center, angle):

NotImplementedError: 

```

Reply:
Ok, this would need to be implemented in `regions`. So we just keep as is for now...